src/scheduler/node_info.c

```diff
@@ -2887,7 +2886,7 @@ eval_simple_selspec(status *policy, chunk *chk, node_info **pninfo_arr,
 	resource_req	*ncpusreq = NULL;
 	resource_req	*aoereq = NULL;
 	resdef		*ncpusdef = find_resdef(consres, "ncpus");
-	resdef		*aoedef = find_resdef(allres, "aoe");
+	resdef		*aoedef = getallres(RES_AOE);
```

Comment:
Please remove ncpusdef and aoedef and replace the uses of them with getallres(RES_NCPUS) and getallres(RES_AOE)

Reply:
Done.